### PR TITLE
NavigationViewTemplateSettings.OpenPaneLength API Spec

### PR DESCRIPTION
**Pull request by gabbybilka**
API spec for NavigationViewTemplateSettings.OpenPaneLength.

### CODE REVIEW

**Review by oldnewthing — APPROVED** [review]

File: specs/NavigationViewTemplateSettings-OpenPaneLength.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+there is a Xaml error when the TemplateSettings property is not public.
+See [issue #6682](https://github.com/microsoft/microsoft-ui-xaml/issues/6682)
+for more information on the error caused by non-public TemplateSettings.
+
+# API Pages
+
+## NavigationViewTemplateSettings.OpenPaneLength
+
+Gets the `min` between `OpenPaneLength` and the window's width.
+This is the calculated value of the width of the pane when opened and fully expanded. Defaults to 320.0.
```
Comment (david-risney) [inline specs/NavigationViewTemplateSettings-OpenPaneLength.md:47]: Should we document the units too? Or is it generally assumed for all these sort of properties that they're DIPs and we don't need to explicitly say that?

Comment (Karl-Bridge-Microsoft) [reply specs/NavigationViewTemplateSettings-OpenPaneLength.md:47]: Units are an important detail and should be specified.

File: specs/NavigationViewTemplateSettings-OpenPaneLength.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
+NavigationViewTemplateSettings.OpenPaneLength
```
Comment (Karl-Bridge-Microsoft) [inline specs/NavigationViewTemplateSettings-OpenPaneLength.md:1]: For the most part, I think this is a pretty good example draft. It contains most everything we'd need to pop a corresponding API ref topic. Is it possible to review the raw template? There are a few small issues in this draft that don't make it directly portable to a ref doc. This is fine if PMs hand a spec off to writers to pop the topics as we can pull the relevant details and fit them in to the correct sections of the topic (for example, the property description specifies the default, which would typically be specified in the property type description). It's important to keep the main description as concise as possible due to it being consumed for index topics (class/namespace/etc) and intellisense.

Comment (Karl-Bridge-Microsoft) [reply specs/NavigationViewTemplateSettings-OpenPaneLength.md:1]: This particular example was used to directly pop a ref topic, which then needed the refinements described. So some duplication of effort (create spec, pop ref topic with info in spec, writer review comments suggesting the necessary revisions, revisions completed, re-review, etc).

Comment (MikeHillberg) [reply specs/NavigationViewTemplateSettings-OpenPaneLength.md:1]: > Is it possible to review the raw template

Yes! Let's open a separate Issue (or PR) on https://github.com/microsoft/WindowsAppSDK/blob/main/specs/spec_template.md.

File: specs/NavigationViewTemplateSettings-OpenPaneLength.md (hunk 1 above)
Comment (gabbybilka) [reply specs/NavigationViewTemplateSettings-OpenPaneLength.md:47]: Added 'in view pixels' for this spec. Also making similar update in the ref topic.

**Comment by gabbybilka** [comment]
@Karl-Bridge-Microsoft @MikeHillberg @david-risney Any additional comments or ready to merge?

File: specs/NavigationViewTemplateSettings-OpenPaneLength.md (hunk 1 above)
Comment (Karl-Bridge-Microsoft) [reply specs/NavigationViewTemplateSettings-OpenPaneLength.md]: Yes, the ref topic should have the same detail.

**Comment by gabbybilka** [comment]
> @Karl-Bridge-Microsoft @MikeHillberg @david-risney Any additional comments or ready to merge?

Cleaning up old PRs, can we merge in?

**Review by bpulliam — APPROVED** [review]

**Pull request merged**